packages/react-components/src/components/UnsupportedOperatingSystem.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
```

Comment:
these all shouldn't be needed. You can create the pure ui component but just conditional compile the export line